src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/toolkit/taglets/TagletWriter.java

```diff
@@ -60,186 +61,185 @@ protected TagletWriter(boolean isFirstSentence) {
     }
 
     /**
-     * @return an instance of an output object.
+     * Returns an instance of an output object.
+     *
+     * @return an instance of an output object
      */
     public abstract Content getOutputInstance();
 
     /**
-     * Return the output for a {@code {@code ...}} tag.
+     * Returns the output for a {@code {@code ...}} tag.
+     *
+     * @param element The element that owns the doc comment
+     * @param tag     the tag
      *
-     * @param element
-     * @param tag the tag.
-     * @return the output of the taglet.
+     * @return the output
      */
     protected abstract Content codeTagOutput(Element element, DocTree tag);
 
     /**
-     * Return the output for a {@code {@index...}} tag.
+     * Returns the output for a {@code {@index...}} tag.
+     *
+     * @param element The element that owns the doc comment
+     * @param tag     the tag
      *
-     * @param tag the tag.
-     * @return the output of the taglet.
+     * @return the output
      */
     protected abstract Content indexTagOutput(Element element, DocTree tag);
 
     /**
-     * Returns the output for the DocRoot inline tag.
-     * @return the output for the DocRoot inline tag.
+     * Returns the output for a {@code {@docRoot}} tag.
+     *
+     * @return the output
      */
     protected abstract Content getDocRootOutput();
 
     /**
-     * Return the deprecated tag output.
+     * Returns the output for a {@code @deprecated ...} tag.
```

Comment:
In this file "ellipsis" is used for inline tags.

Reply:
OK.